docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -724,6 +725,37 @@ directly or used in conditional statements.
 - `$file_size`
   : Size of the file for this row.
 
+### Replace tables
+
+The connector supports replacing a table, in case the table already exists, as
+an atomic operation. Atomic table replacement creates a new snapshot with the
+new table definition and keeps the existing table history. See
+{ref}`delta-lake-history-table` in metadata tables.
```

Comment:
use markdown syntax for ref link `[](delta-lake-history-table)` .. and optionally with title